bin/exec.go

```diff
@@ -146,7 +146,7 @@ func main() {
 		experiment.JobName = engineDetails.Experiments[i] + "-" + randomString
 
 		log.Infof("JobName for this Experiment : %v", experiment.JobName)
-
+		analytics.TriggerAnalytics(engineDetails.Experiments[i])
```

Comment:
place a comment here saying triggering  event before beginning job run

Reply:
inserted the comment @ksatchit